src/features/click-to-load.js

```diff
@@ -2016,16 +2016,16 @@ export default class ClickToLoad extends ContentFeature {
     get messaging () {
         if (this._messaging) return this._messaging
 
-        if (this.platform.name === 'android' || this.platform.name === 'extension' || this.platform.name === 'macos') {
+        if (this.platform.name === 'android' || this.platform.name === 'extension') {
             this._clickToLoadMessagingTransport = new SendMessageMessagingTransport()
             const config = new TestTransportConfig(this._clickToLoadMessagingTransport)
             this._messaging = new Messaging(this.messagingContext, config)
             return this._messaging
-        } else if (this.platform.name === 'ios') {
+        } else if (this.platform.name === 'ios' || this.platform.name === 'macos') {
             const config = new WebkitMessagingConfig({
                 secret: '',
                 hasModernWebkitAPI: true,
-                webkitMessageHandlerNames: ['contentScopeScripts']
+                webkitMessageHandlerNames: ['contentScopeScriptsIsolated'] // contentScopeScriptsIsolated
```

Comment:
We're not going to be in the isolated context are we? IMO this should be changed back.

```suggestion
                webkitMessageHandlerNames: ['contentScopeScripts']
```

Reply:
I meant to remove that comment, but we do require `contentScopeScriptsIsolated`.  The issue is that the `subscribe` messaging pattern didn't work otherwise, @shakyShane  and I spent some time looking at that but this appeared to be the best path forward.  CTL works fine in isolated FWIW, and seems like the safer path anyway?

Reply:
Yeah cool just didn't think it did work, just remove the comment instead and we're golden. Thanks for the explanation :)

Reply:
@jonathanKingston oh yep, I removed the comment already so resolving this thread.